README.md

```diff
@@ -36,24 +35,23 @@ This node will require the following AWS account IAM role permissions:
 - `cloudwatch:PutMetricData`
 
 ### Building from Source
-Create a ROS workspace and a source directory
 
-    mkdir -p ~/ros-workspace/src
+To build from source you'll need to create a new workspace, clone and checkout the latest release branch of this repository, install all the dependencies, and compile. If you need the latest development features you can clone from the `master` branch instead of the latest release branch. While we guarantee the release branches are stable, the `master` should be considered to have an unstable build due to ongoing development. 
```

Comment:
Add a hard to miss disclaimer about using Master branch .

Reply:
I've highlighted the note that the master branch should not be considered stable. There's also a new note in the dependencies update section calling out that building master may require manual builds of dependencies too.